proto/buf/alpha/registry/v1alpha1/webhook_payload.proto

```diff
@@ -0,0 +1,52 @@
+// Copyright 2020-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.alpha.registry.v1alpha1;
```

Comment:
This doesn't belong in registry, registry so far is the services that make up a buf schema registry. This is something a customer would implement, it should probably go into its own package?

Reply:
`buf.alpha.registry.v1alpha1.webhook`? Package level separation seems nice here so that if we ever have other interfaces for the user to implement that we call out to, they won't be forced to drag in the webhook types if they don't need to.

Reply:
The last component of a package should be a version as per [the style guide](https://docs.buf.build/best-practices/style-guide#files-and-packages), might be better to just separate it from `registry` and be `buf.alpha.webhook.v1alpha1`?

Reply:
Done, new `webhook` package.